src/containers/popup.jsx

```diff
@@ -0,0 +1,41 @@
+// Using leaflet API for binding popup due to using <GeoJson />.
+// Creator says "it is unnecessarily complex" to use <Popup /> in our use.
+// Reference:https://github.com/PaulLeCam/react-leaflet/issues/33
+
+const popupOptions = {
+  maxWidth: 'auto',
+  maxHeight: '300',
+  className: 'leaflet_popup',
+};
+
+const parseGeoJson = data =>
+  Object.entries(data).map(([key, value]) => {
```

Comment:
Suggestion on how to rewrite geojson mapping to handle arrays (prints keyname for first array entry and blank for the others):

```
const parseGeoJson = data =>
  Object.entries(data).map(([key, value]) => {
    if (value) {
      if (Array.isArray(value)) {
        return value.reduce((accumulator, element, currentIndex) => {
          const keyValue = currentIndex === 0 ? key : '&nbsp;';
          accumulator += `<tr><td>${keyValue}</td><td>${element}</td></tr>`; // eslint-disable-line no-param-reassign
          return accumulator;
        }, '');
      }
      return typeof value === 'object'
        ? parseGeoJson(value)
        : `<tr>
          <td>${key}</td>
          <td>${value}</td>
          </tr>`;
    }
    return `<tr>
            <td>${key}</td>
            <td>unknown</td>
            </tr>`;
  });

```

Reply:
I like this. Implemented it and added formatting to the key strings.

Reply:
Still trying to figure out where the commas are coming in from. There are 29 if that has any significance to you in correlation with the data.